docker-compose.yml

```diff
@@ -27,9 +29,12 @@ services:
     ports:
       - '3001:3001'
     depends_on:
-      - valkey
-      - db
-      - blobstorage
+      db:
+        condition: service_healthy
```

Comment:
Build was failing bringing up the docker compose because the backend image was trying to run migrations before the postgres image was ready.